erpnext/accounts/report/financial_statements.py

```diff
@@ -258,3 +258,35 @@ def get_columns(period_list):
 		})
 
 	return columns
+
+
+def get_data_account_type(company, account_type, period_list):
+    data = {}
+    for period in period_list:
+        gl_sum = frappe.db.sql_list("""select sum(credit) - sum(debit) FROM `tabGL Entry`
+                                    where company='%s' and posting_date >= '%s' and posting_date <= '%s' and
+                                    account in ( SELECT name FROM tabAccount WHERE account_type = '%s')""" %
+                                    (company, period['from_date'], period['to_date'], account_type))
```

Comment:
You can do it if you find some time. Else Nabin will look into it on Monday. 

Sent from my phone

> On 12-Dec-2015, at 10:01 AM, Chris Ian Fiel <notifications@github.com> wrote:
> 
> In erpnext/accounts/report/financial_statements.py:
> 
> > @@ -258,3 +258,35 @@ def get_columns(period_list):
> >  		})
> >  
> >  	return columns
> > +
> > +
> > +def get_data_account_type(company, account_type, period_list):
> > +    data = {}
> > +    for period in period_list:
> > +        gl_sum = frappe.db.sql_list("""select sum(credit) - sum(debit) FROM `tabGL Entry`
> > +                                    where company='%s' and posting_date >= '%s' and posting_date <= '%s' and
> > +                                    account in ( SELECT name FROM tabAccount WHERE account_type = '%s')""" %
> > +                                    (company, period['from_date'], period['to_date'], account_type))
> @anandpdoshi oh i see do i need to change this? or can you do it in your side :)
> 
> —
> Reply to this email directly or view it on GitHub.
>